media/docs/build/building_with_sycl_support.md

```diff
@@ -1,103 +1,134 @@
 
-# Running CMake
+Only Linux platforms are supported.
 
-## Required CMake options
+### Requirements (SYCL for Intel GPU)
 
-The SYCL build requires specifying the following CMake options and
-environmental variables. Replace `<path-to-clang>` and `<path-to-clang++>`
-with the path to your clang and clang++ executables. You may use `clang`
-and `clang++` directly if they are in your `PATH`.
+To build CUTLASS SYCL support for Intel GPUs, you need the Intel SYCL compiler,
+you can use the latest [[nightly build](https://github.com/intel/llvm/releases)] 
+or a oneAPI toolkit from 2025.0 onwards.
 
-```bash
-$ export CC=<path-to-clang>/clang
-$ export CXX=<path-to-clang++>/clang++
+Building the tests and the examples requires oneMKL for random number generation.
+
+### Building with SYCL for Intel GPU support
+
+The following instructions show how to use the nightly build to build the cutlass example
+
+
+```
+# Download the nightly of DPCPP compiler
+$ wget https://github.com/intel/llvm/releases/tag/nightly-2025-01-31
+# Setup the environment variables
+$ export PATH_TO_DPCPP=/path/to/your/dpcpp/installation
+$ export PATH=${PATH_TO_DPCPP}/bin/:$PATH
+$ export LD_LIBRARY_PATH=${PATH_TO_DPCPP}/lib/:$LD_LIBRARY_PATH
+$ export RPATH=${PATH_TO_DPCPP}/lib/:$RPATH
+# Create the build directory and configure CMake
+# mkdir build_intel; cd build_intel
+$ CC=clang CXX=clang++ cmake .. -G Ninja \
+  -DCUTLASS_ENABLE_SYCL=ON \
+  -DDPCPP_SYCL_TARGET=intel_gpu_pvc \
+  -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+  -DCMAKE_CXX_FLAGS="-ftemplate-backtrace-limit=0 -fdiagnose-colors=always"  
 ```
 
-### CMake options for Nvidia devices
+CMake will check that DPC++ compiler is available in the system,
+and it will download MKL library if it cannot find it.
 
-* `DPCPP_SYCL_TARGET=nvptx64-nvidia-cuda` sets the triplet target for Nvidia GPUs.
-* `DPCPP_SYCL_ARCH=sm_80` sets the device architecture to SM_80.
+To build and run a simple PVC gemm example run the commands below.
 
-```bash
-$ cmake -G Ninja .. \
-    -DCUTLASS_ENABLE_SYCL=ON \
-    -DDPCPP_SYCL_TARGET=nvptx64-nvidia-cuda \
-    -DDPCPP_SYCL_ARCH=sm_80
+```
+$ ninja examples/sycl/pvc/pvc_gemm
+$ cd examples/sycl/pvc/
+$ ./pvc_gemm
+Disposition: Passed
+Problem Size: 5120x4096x4096x1
+Cutlass GEMM Performance:     [247.159]TFlop/s  (0.6951)ms
 ```
 
-Building Cutlass with SYCL support on Nvidia devices has been tested
-on an A100 device with `DPCPP_SYCL_ARCH=sm_80`.
+## Support for NVIDIA GPUs (validation only)
 
-### CMake options for Intel devices
+The SYCL backend supports compilation for NVIDIA GPUs using the 
+oneAPI NVIDIA plugin. This support is only for testing and validation
+purposes and not intended for production.
 
-* `DPCPP_SYCL_TARGET=intel_gpu_pvc` sets the triplet target for Intel PVC GPUs.
+### Requirements 
+ 
+To build CUTLASS SYCL support you need the latest version of DPC++ compiler. You can either use a recent [nightly build](https://github.com/intel/llvm/releases)
```

Comment:
Here you refer to the DPC++ compiler, but above it's the Intel SYCL compiler. Might be a little confusing